Packs/Okta/ReleaseNotes/2_1_4.md

```diff
@@ -0,0 +1,4 @@
+
+#### Classifiers
+##### User Profile - Okta (Outgoing)
+- Maintenance and stability enhancements.
```

Comment:
Here and everywhere else, let's avoid using this line and instead write down what actually happened

Reply:
done